tests/orders/tests-edd-payment.php

```diff
@@ -911,6 +911,26 @@ public function test_get_tax_rate_returns_decimal() {
 		$this->assertEquals( 0.2, $payment->tax_rate );
 	}
 
+	public function test_receipt_show_download_files() {
+		$payment      = $this->payment;
+		$receipt_args = array(
+			'id' => $payment->ID,
+		);
+		$cart         = edd_get_payment_meta_cart_details( $payment->ID, true );
+		$cart_item    = reset( $cart );
+		$download_id  = $cart_item['id'];
+
+		// Test sending a payment item array to the filter, which will convert it to an \EDD\Orders\Order_Item object.
+		add_filter( 'edd_order_receipt_show_download_files', function( $ret, $item_id, $order_receipt_args, $order_item_object ) use ( $cart_item ) {
+			$this->assertTrue( $order_item_object->id === $cart_item['order_item_id'] );
```

Comment:
```suggestion
			$this->assertInstanceOf( '\\EDD\\Orders\\Order_Item', $order_item_object );
			$this->assertTrue( $order_item_object->id === $cart_item['order_item_id'] );
```

Just to add in one additional check. It's a little redundant, perhaps, but. :shrug: